not/localization/__init__.py

```diff
@@ -0,0 +1,98 @@
+"""The localization module provides localized user-facing strings
+
+generally, one need only import the polyglot singleton object
+
+This module uses an unorthodox approach to singletons to avoid keeping
+state in the module, which feels more awkward.
+
+When python 3.9 comes out in earnest, revisit using:
+https://www.python.org/dev/peps/pep-0603/
+"""
+
+import ctypes
+from json import load
+from locale import getdefaultlocale, normalize, windows_locale
+from os import getenv
+from pathlib import Path
+from platform import system
+from typing import Dict, Union
+
+DEFAULT_LOCALE = normalize("en")
+STRINGS_SUPPORTED = set("")
```

Comment:
did `set("")` in lieu of type hinting this. It wont be necessary to hint it once actual strings are in there.